src/EnergyPlus/UserDefinedComponents.cc

```diff
@@ -114,7 +114,6 @@ namespace UserDefinedComponents {
 
     void UserPlantComponentStruct::onInitLoopEquip(EnergyPlusData &state, const PlantLocation &calledFromLocation)
     {
-        bool anyEMSRan;
```

Comment:
[src/EnergyPlus/UserDefinedComponents.cc:117]:(style),[variableScope],The scope of the variable 'anyEMSRan' can be reduced.